### PR TITLE
docs/variants/novacustom_nv4x/building.md: make build manual consiste…

### PR DESCRIPTION
**Pull request by mkopec**
…nt with ns5x

Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

**Comment by mkopec** [comment]
@macpijan This fixes the GitHub credentials error related to cloning the submodules.

File: docs/variants/novacustom_nv4x/building.md [hunk 1]
```diff
@@ -21,48 +21,36 @@ Obtain the image:
 docker pull coreboot/coreboot-sdk:0ad5fbd48d
 ```
 
-Obtain coreboot source code for NovaCustom NV4X:
+Clone the coreboot repository:
 
 ```bash
-git clone https://github.com/Dasharo/coreboot.git
+git remote add dasharo https://github.com/dasharo/coreboot.git
```
Comment (macpijan) [inline docs/variants/novacustom_nv4x/building.md:27]: @mkopec The clone is entirely removed. Intentional?

**Review by macpijan — CHANGES_REQUESTED** [review]
The clone command is entirely removed

File: docs/variants/novacustom_nv4x/building.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/novacustom_nv4x/building.md]: no, sorry. fixed

File: docs/variants/novacustom_nv4x/building.md [hunk 2]
```diff
@@ -21,48 +21,36 @@ Obtain the image:
...
-git clone https://github.com/Dasharo/coreboot.git
+git clone https://review.coreboot.org/coreboot.git
 ```
 
 Navigate to the source code directory and checkout to the desired revision:
 
 > Replace the REVISION with one of the:
 > - `novacustom_nv4x/release` for the latest released version
-> - `novacustom_nv4x/vVERSION` (e.g. `v1.0.1`) for the given release
+> - `novacustom_nv4x/vVERSION` (e.g. `v1.2.1`) for the given release
```
Comment (macpijan) [inline docs/variants/novacustom_nv4x/building.md:34]: @mkopec This is not correct tag format

Comment (mkopec) [reply docs/variants/novacustom_nv4x/building.md]: Fixed

**Review by macpijan — APPROVED** [review]

**Pull request merged**